integration/test/ca_bundle/ca_bundle_test.go

```diff
@@ -0,0 +1,114 @@
+package ca_bundle
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"testing"
+	"time"
+)
+
+const configOutputPath = "/opt/aws/amazon-cloudwatch-agent/bin/config.json"
+const commonConfigOutputPath = "/opt/aws/amazon-cloudwatch-agent/etc/common-config.toml"
+const configJSON = "/config.json"
+const commonConfigTOML = "/common-config.toml"
+const outputLog = "/opt/aws/amazon-cloudwatch-agent/logs/amazon-cloudwatch-agent.log"
+const targetString = "x509: certificate signed by unknown authority"
+//Let the agent run for 1 minutes. This will give agent enough time to call server
+const agentRuntime = 1
+
+
+type input struct {
+	findTarget bool
+	dataInput string
+}
+
+//Must run this test with parallel 1 since this will fail if more than one test is running at the same time
+func TestBundle(t *testing.T) {
+
+	parameters := []input{
+		{dataInput: "resources/integration/ssl/with/combine/bundle", findTarget: false},
+		{dataInput: "resources/integration/ssl/with/original/bundle", findTarget: true},
+		{dataInput: "resources/integration/ssl/without/bundle", findTarget: true},
+	}
+
+	for _, parameter := range parameters {
+		//before test run
+		log.Printf("resource file location %s find target %t", parameter.dataInput, parameter.findTarget)
+		clearLogFile()
+		t.Run(fmt.Sprintf("resource file location %s find target %t", parameter.dataInput, parameter.findTarget), func(t *testing.T) {
+			copyFile(parameter.dataInput + configJSON, configOutputPath)
+			copyFile(parameter.dataInput + commonConfigTOML, commonConfigOutputPath)
+			startTheAgent();
+			time.Sleep(agentRuntime * time.Minute);
+			log.Printf("Agent has been running for : %d", agentRuntime);
```

Comment:
I initially had a gripe that we're storing a plain int in `agentRuntime`, but got over it because I think it's a typical use case to do `num * time.Unit` for a duration, however here we're logging that the agent has been running for n... without the unit of measure. And I don't like the idea of hardcoding the string `"minute(s)"` into the log if we don't have to (could accidentally miss that if we change the sleep time later). 

Would you be opposed to initializing a duration before the sleep so we can reference it in the print log after?
```
d := agentRuntime * time.Minute
time.Sleep(d);
log.Printf("Agent has been running for %s, d.String());
```
or something like that - though, I think it might just be simpler to change `agentRuntime` to be the duration value itself.